### PR TITLE
Bfix(Twitter wall): Fixes size issue of horizontal images and tweet-frequency card

### PR DESCRIPTION
**Pull request by sopu**

### CODE REVIEW

**Comment by sopu** [comment]
Fixes #520

**Comment by aneeshd16** [comment]
@sopu can you please show a screenshot?

**Comment by sopu** [comment]
@aneeshd16 
Please look at the following screenshots
1. Full screen
screenshot from 2016-02-27 11 56 49

2. Smaller screen but still showing leaderboard and map.
screenshot from 2016-02-27 11 57 03

3. Still smaller. 
screenshot from 2016-02-27 11 57 19

**Comment by aneeshd16** [comment]
@sopu looks good. In the third screenshot, the header is pushed to the left. Any ideas on how to solve this?

**Comment by sopu** [comment]
It is due to the media properties that change when width < 992px. 
screenshot from 2016-02-27 12 08 08

**Comment by sopu** [comment]
@aneeshd16 Now check. 
screenshot from 2016-02-27 12 32 30
screenshot from 2016-02-27 12 32 02

**Comment by aneeshd16** [comment]
@sopu Awesome. Possible to decrease the font size of the leaderboard tabs when screen size is reduced? It will be better if they are in the same row.

Also, I do not think this PR solves #520. That issue is concerned with utilizing vertical space for images displayed in the wall. However, this is a good patch, solves major responsiveness issues for the wall.

**Comment by sopu** [comment]
@aneeshd16. It does. 
Refer: https://github.com/sopu/loklak_webclient/commit/496917a4fbb9682e1a31b62b61a28ba50cbf3376

**Comment by sopu** [comment]
screenshot from 2016-02-27 13 04 23
Fixed.

**Comment by aneeshd16** [comment]
this is great. Will test and merge tomorrow.

**Comment by sudheesh001** [comment]
@sopu Please squash the commits into one and mention the `Fixes #520` in the commit message itself.

**Comment by sopu** [comment]
@sudheesh001  done :+1:

File: app/styles/_admin.scss [hunk 1]
```diff
@@ -2558,7 +2558,8 @@ a:focus {
 .nav-tabs-custom > .nav-tabs > li {
   border-top: 3px solid transparent;
   margin-bottom: -2px;
```
Comment (sudheesh001) [inline app/styles/_admin.scss]: Since you're using `%` for the sake of uniformity, can you replace the other `px` values with the `%` too ?

Comment (sopu) [inline app/styles/_admin.scss]: Thanx

**Comment by sopu** [comment]
@sudheesh001 I have fixed the **px -> %** for tags where I have made changes. 
The other places that require such modifications, I am leaving them untouched as they don't concern this issue.

**Comment by aneeshd16** [comment]
Okay:
image

Still okay: 
image

Reduced the size a little bit more, leaderboard goes for a toss: 
image

There are 4 cases:
1. Full version, everything displayed properly - OK
2. Reduced screen width, everything displayed properly - OK
3. Reduced more, leaderboard messes up **(need to fix)** (Possible solution: Hide the leaderboard and stats here, instead of in the next case)
4. Reduced more, leaderboard and stats are hidden - OK

@sopu What do you think?

**Comment by sopu** [comment]
@aneeshd16 Not able to reproduce the issue. 
( _3. Reduced more, leaderboard messes up (need to fix) (Possible solution: Hide the leaderboard and stats here, instead of in the next case_ )

**Comment by aneeshd16** [comment]
@sopu, as the width of the window is reduced, the width of the graph also reduces. Accordingly, the height of the graph also reduces. This "squishes" the y axis (see last pic in previous comment).
IMO this PR has other imp fixes as well, lets merge this, then we can track this bug and fix it separately. What do you think @sudheesh001 ?

**Comment by sopu** [comment]
@aneeshd16: I agree :+1: 
I will check that issue.

**Comment by sudheesh001** [comment]
I agree with that @aneeshd16 Feel free to merge this to :smile:

**Pull request merged**